buildscripts/macos-setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#verify python version
+PYTHONVERSION=$(python -V 2>&1 | grep -Po '(?<=Python )(.+)');
+if [[ -z "$PYTHONVERSION" ]]; then
+    echo "No Python version found, exiting setup ... ";
+    exit 1;
+elif [[ ${PYTHONVERSION:0:1} -ne 3 || ${PYTHONVERSION:2:1} -lt 6 ]]; then
+    echo "Wrong Python version installed. requires >= Python 3.6";
+    exit 1;
+fi
+
+#TODO add more user setup verification
+
+## install windows requirements
+(cd buildscripts && pip install -r requirements.txt)
```

Comment:
are they going to run this from the buildscripts folder? or will they have to run ` ./buildscripts/macos-setup.sh`. same question for the windows one

Reply:
the script will be executed form project root as `./buildscripts/XXXX-setup.sh` the directory change for the pip install is managed within the script